### PR TITLE
Memoized random

### PR DESCRIPTION
**Pull request by naddah-5**

### CODE REVIEW

File: src/ai_rest.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def predict_on_images(images):
         "shape": images[0].shape
     })
     request_headers = {"Content-Type": "application/json"}
-    request_url = "https://pepper.lillbonum.se/predict/hand"
+    request_url = "http://host.docker.internal:5000/"
```
Comment (Rickos99) [inline src/ai_rest.py:31]: I think its better to keep the url to external server

File: src/rps_server_rest.py [hunk 2]
```diff
@@ -40,7 +40,7 @@ def make_http_request_to_ai(image):
         "shape": [1, 100, 100, 1]
     })
 
-    URL = "https://pepper.lillbonum.se/predict/hand"
+    URL = "http://host.docker.internal:5000/predict/hand"
```
Comment (Rickos99) [inline src/rps_server_rest.py:43]: I think its better to keep the url to external server

**Review by Rickos99 — CHANGES_REQUESTED** [review]
I left some comments

File: src/ai_rest.py (hunk 1 above)
Comment (naddah-5) [reply src/ai_rest.py]: fixed

File: src/rps_server_rest.py (hunk 2 above)
Comment (naddah-5) [reply src/rps_server_rest.py]: fixed

**Pull request merged**